.github/workflows/ci.yaml

```diff
@@ -224,6 +224,37 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  mutation-test:
+    name: Mutation Testing
+
+    if: github.event_name == 'pull_request'
```

Comment:
It should be run on push to `main` and release bugfix branches as well - base commit is the prev commit

Reply:
the current plan is to start slowly and only in pull requests